### ISSUE TITLE
MA refuses playing to renderer

### ISSUE THREAD

**Issue by waarsenburg** [music-assistant/support#3240]
### What version of Music Assistant has the issue?

2.3.3

### What version of the Home Assistant Integration have you got installed?

2024.11.4

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

I have 2 players: MusicBox running on a Pi Zero and a Technisat 143 radio/media player. The musicbox works, but the Technisat doesn't. It DOES work when playing a single song from Media, but MA seems to start playing and then stops after 2 seconds or so.

### How to reproduce

Choose Music Assistant from the side bar, select DIgitradio as renderer, select album and hit Play

### Music Providers

Music Assistant, Filesystem (Local Disk)

### Player Providers

UPnP/DLNA Player Provider

### Full log output

[d5369777_music_assistant_2024-11-28T16-10-37.125Z.log](https://github.com/user-attachments/files/17950426/d5369777_music_assistant_2024-11-28T16-10-37.125Z.log)

### What version of Home Assistant Core are your running

2024.11.3

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Raspberry Pi

**Comment by waarsenburg** [comment]
The Technisat works with all my other streamers: BubblePnP and Universal Media Serveri.

**Comment by waarsenburg** [comment]
Sorry, accidentally closed...

**Comment by OzGav** [comment]
Please follow the troubleshooting page closely. You have not listed what you have tried to resolve this yourself particularly the requirements on https://music-assistant.io/player-support/dlna/

**Comment by waarsenburg** [comment]
I tried every possible combination of settings. Currently, what works (more or less) is Disable Queue flow mode, automatically play,  disable volume normalization, disable (lossy) mp3 stream, Profile 3 - forced content length, disable Pre-announce TTS announcements. Now in most cases, MA starts playing but there are 3 issues: sometimes MA stops streaming after the first track, sometimes it starts the next track on 80% of the previous track (so, without letting it finish playing), or it will crash and the Technisat renderer/radio switches back to the last selected source before MA took over. Media provider is Local Disk (an SMB mount to a Linux server containing the mp3's).

**Comment by OzGav** [comment]
OK thanks. You probably saw this then

> Unfortunately due to the difficulty in resolving these issues they are low priority. If your device supports a different protocol then use that instead of raising an issue

If you can't use any other provide type with the device then there is not much more we can do at this time.

When you say "it will crash" do you mean the whole MA server goes down and has to be restarted or do you just mean the stream stops?

**Comment by grainsoflight** [comment]
@OzGav I am also experiencing an issue with DLNA on a specific device. Tried all profiles and tbh, may just abandon that device altogether since DLNA seems to have issues, unless I am missing something. The only error it logs related to the stream is 

2024-12-02 20:33:05.048 ERROR (MainThread) [music_assistant.dlna] Error during call cmd_pause: 'Error during async_call(), action: Pause, status: 500, upnp error: 501 (Action Failed)'

Not really sure how to interpret. Does that look like something related to the profiles or do you think I have a problem elsewhere?

**Comment by OzGav** [comment]
Yes probably a device which requires a particular tweak to fix which we don't have the capacity to do.

**Comment by grainsoflight** [comment]
okie dokey

**Comment by jonathanarchergithub** [comment]
Hi, knowing this has low prio, just wanted to add some more info.
I am having a very similar problem as [waarsenburg](https://github.com/waarsenburg) describes with a Technisat Multyradio 4.0. I think in my case the problem is only linked to the queue flow mode option. If I enable it, DLNA streaming works (but stops after one song of course), if I enable it, it will not play.

Funny enough, using the Home Assistant player provider for this device (DMR Integration) it works fine even continuing to play the songs of an album (so apparently the device does support queuing although in the MA logs it says otherwise).

**Comment by OzGav** [comment]
@jonathanarchergithub have you tried each of the different http profiles with queue flow mode on?

**Comment by jonathanarchergithub** [comment]
@OzGav yes. There's no difference in behaviour. Btw. I notice the stereo displays "buffering" with a progress bar when I hit play but the progress bar stays at 0%.

**Comment by OzGav** [comment]
@jonathanarchergithub then unfortunately I direct your attention to the docs and these points

> Due to the very inconsistent implementation of this protocol by manufacturers, some players will work great and others will simply not work at all or need workarounds.

> Unfortunately due to the difficulty in resolving these issues they are low priority. If your device supports a different protocol then use that instead of raising an issue

**Comment by jonathanarchergithub** [comment]
I know, that's okay. Just wanted to add the info. I noticed a typo in my post however. I meant that it works with flow mode disabled, it doesn't play at all (and shows the 0% progress bar) with flow mode enabled. The HA player provider version (via DMR integration) works well via MA and plays all songs of an album.

**Comment by OzGav** [comment]
So many chnges in MA with the release of 2.4 so @waarsenburg can you confirm this is still an issue with 2.4.4?

**Comment by jonathanarchergithub** [comment]
At least for my players I can confirm that it works much better now, for some reason now it even plays correctly without Flow Mode. MA doesn‘t complain that the player doesn‘t support enqueuing anymore (though on FW change on player side). So thumbs up from me:-)

**Comment by OzGav** [comment]
Closing assuming fixed. Can reopen if necessary
